### PR TITLE
[supervisor-api] fix generate.sh script

### PR DESCRIPTION
**Pull request by akosyakov**
and add missing copyrights

### CODE REVIEW

**Review by csweichel — APPROVED** [review]
LGTM

File: chart/templates/server-proxy-ssl-dhparam.yaml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Copyright (c) 2021 Gitpod GmbH. All rights reserved.
```
Comment (AlexTugarev) [inline chart/templates/server-proxy-ssl-dhparam.yaml:1]: @akosyakov, please revert this change, or fix the build otherwise.

@corneliusludmann, thank you for figuring this out! 🙏🏻
